source/shared/support/_dynamic-search-bar.html.erb

```diff
@@ -0,0 +1,29 @@
+<script>
+  var filterInput = document.getElementById('search-table');
+  filterInput.addEventListener('keyup', filterResults);
+
+  function filterResults() {
+    var filterValue = filterInput.value.toUpperCase();
+    var rows = document.querySelectorAll('.result-row');
+
+    for(var i = 0; i < rows.length; i++) {
+      var text = rows[i].querySelectorAll('.govuk-table__cell')[0].innerHTML.toUpperCase();
```

Comment:
I would keep a dedicated selector for targetting this text, this class is already used for styling.